website/docs/crud.md

```diff
@@ -0,0 +1,293 @@
+---
+id: crud
+title: CRUD Handler
+sidebar_label: CRUD Handler
+---
+
+## CRUD
+`Core-Lib` provides `CRUD ( Create, Read, Update, Delete )` `DataAccess` classes as boilerplate code.
+
+They are as follows
+
+
+
+### CRUD
+This class is used to initialize `CRUD` with database objects, rule validators and database handlers. `CRUD` is an abstract base class for all types of `CRUDDataAccess` in `Core-Lib`.
+```python
+class CRUD(ABC):
+
+    def __init__(self, db_entity, db: SqlAlchemyDataHandlerRegistry, rule_validator: RuleValidator):
+        self._db_entity = db_entity
+        self._db = db
+        self._rule_validator = rule_validator
+```
+`db_entity` the database class or object to be passed for initialization.
+
+`db` instance of the `SqlAlchemyDataHandlerRegistry` used to connect to the database.
+
+`rule_validator` rules must be created with `RuleValidator` and used to validate data passed to the `create()` and `update()`.

```

Comment:
this phrase is not clear.   
`RuleValidator` instance used to validate data pass into `create()` and `update()` methods